src/Compilers/CSharp/Portable/Binder/LocalScopeBinder.cs

```diff
@@ -204,8 +204,20 @@ protected ImmutableArray<LocalSymbol> BuildLocals(SyntaxList<StatementSyntax> st
                         {
                             Binder localDeclarationBinder = enclosingBinder.GetBinder(innerStatement) ?? enclosingBinder;
                             var decl = (LocalDeclarationStatementSyntax)innerStatement;
-                            LocalDeclarationKind kind = decl.IsConst ? LocalDeclarationKind.Constant : LocalDeclarationKind.RegularVariable;
-                            foreach (var vdecl in decl.Declaration.Variables)
+                            LocalDeclarationKind kind;
+                            if (decl.IsConst)
+                            {
+                                kind = LocalDeclarationKind.Constant;
+                            }
+                            else if (decl.UsingKeyword != default(SyntaxToken))
+                            {
+                                kind = LocalDeclarationKind.UsingVariable;
+                            }
+                            else
+                            {
+                                kind = LocalDeclarationKind.RegularVariable;
+                            }
+                                foreach (var vdecl in decl.Declaration.Variables)
```

Comment:
1. in general we like a blank like after a }.  
2. this is indented improperly.
3. avoid abbrs when possible.  you can just say "var variableDeclaration"

Reply:
I just accidentally indented a line or two due to copying and pasting code between branches to isolate parsing from the rest of the feature. Anything including `vdecl` is not my code. Should I change it regardless?

Reply:
No.  Once you fix the indentation, just leave the code alone if it isn't necessary to change for your PR. Thanks! :)